src/current/v23.2/ui-networking-dashboard.md

```diff
@@ -25,6 +25,30 @@ The **Networking** dashboard displays the following time series graphs:
 
 Metric: [`sys.host.net.recv.bytes`]({% link {{ page.version.version }}/essential-metrics-self-hosted.md %}#sys-host-net-recv-bytes) Bytes received on all network interfaces since this process started
 
+## Network Packets Received
+
+- In the node view, the graph shows the packets received on all network interfaces since the CockroachDB process started for the node.
```

Comment:
This applies to all instances of that phrase (assuming it is accurate)

Also, should "CockroachDB process" technically be "`cockroach` process" throughout? I think this would be clearer if so?

```suggestion
- In the node view, the graph shows the packets received on all network interfaces since the CockroachDB process started on the node.
```

Reply:
good catches! I updated
- "for the node" to "on the node".
- "CockroachDB process" to "`cockroach` process"